### PR TITLE
Teacher Application: Email copy changes

### PR DESCRIPTION
**Pull request by islemaster**
# Description

Updates copy for emails related to this year's Teacher Application workflow.

## Links

- [spec](https://docs.google.com/document/d/1F4w6dHrpldmCmaSQ-s4VqnCi26M9XuFS8qXiHDKC-K0/edit#)
- [jira](https://codedotorg.atlassian.net/browse/PLC-486)

## Testing story

Manually verified using the Rails mailer preview.

# Screenshots

## Confirmation to teacher

With partner
image

Without partner
image

## Principal approval request

With partner
image

Without partner
image

## Principal approval confirmation (to principal)

With partner
image

Without partner
image

## Principal approval confirmation (to partner)

With partner
image

Without partner
N/A

## Accepted - No cost registration

With partner
image

Without partner
image

## Registration sent

With partner
image

Without partner
image

## Decline

With partner
image

Without partner
image

## Waitlist

With partner
image

Without partner
image

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by agealy — APPROVED** [review]
content looks good to me

File: dashboard/app/views/pd/application/teacher_application_mailer/principal_approval_completed.html.haml [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
 %p
   Thank you for completing the principal approval form.
-  Your teacher’s application is now considered complete and we will complete our review soon.
+  = @application.first_name + "'s"
+  application is now considered complete and we will complete our review soon.
```
Comment (breville) [inline dashboard/app/views/pd/application/teacher_application_mailer/principal_approval_completed.html.haml]: Double complete :)

**Review by breville — APPROVED** [review]

**Comment by breville** [comment]
Related PR: https://github.com/code-dot-org/code-dot-org/pull/31218

File: dashboard/app/views/pd/application/teacher_application_mailer/principal_approval_completed.html.haml (hunk 1 above)
Comment (islemaster) [reply dashboard/app/views/pd/application/teacher_application_mailer/principal_approval_completed.html.haml]: As spec'd :upside_down_face:.  I'll update to "application is now considered complete and we will review it soon."

**Pull request merged**